README.md

```diff
@@ -36,6 +36,27 @@ Instructions to install both are found in [the lila wiki](https://github.com/lic
 
 The mobile application is configured by default to target `http://127.0.0.1:9663` and `ws://127.0.0.1:9664`, so keep these when installing lila.
 
+### Using Lichess Own Dev Server
+
+If you don't want to set up your local Lila server, you can use [lichess dev](https://lichess.dev/) server to run this app.
+Set up the following values as environment variables:
+
+```
```

Comment:
This code block is a bit confusing as it looks like you'd need to define shell env vars, which is actually not working. It would be better to replace it with the actual full `flutter run --dart-define=....` command.

Reply:
Got it. How about this?

### Using Lichess dev server

To use the [Lichess dev](https://lichess.dev/) server to run this app, set up the following values as environment variables in cloned lichess flutter app.

```
flutter run --dart-define=LICHESS_HOST=lichess.dev --dart-define=LICHESS_WS_HOST=socket.lichess.dev
```

Reply:
Got it. How about this?

### Using Lichess dev server

To use the [Lichess dev](https://lichess.dev/) to run this app, run the following command to set up the Lichess host URLs in the app.

```
flutter run --dart-define=LICHESS_HOST=lichess.dev --dart-define=LICHESS_WS_HOST=socket.lichess.dev
```